scripts/upload-apk.sh

```diff
@@ -12,7 +12,9 @@ if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/badge
     exit 0
 fi
 
-./gradlew bundleRelease
+if [[ "$TRAVIS_BRANCH" != "$PUBLISH_BRANCH" ]]; then
```

Comment:
We always want to build aab

Reply:
great, I've removed the unsigned and unaligned conditions as well as they are not being created